rebar.config

```diff
@@ -1,10 +1,7 @@
-%%-*- mode: erlang -*-
+{require_min_otp_vsn, "18"}.
 
-{deps, [
-        {cowboy, ".*", {git, "git://github.com/extend/cowboy.git", {tag, "0.9.0"}}},
-        {jiffy, ".*", {git, "git://github.com/davisp/jiffy.git", {tag, "0.8.5"}}},
```

Comment:
Naive question.. what is the effect of removing jiffy here? I am using a combination of leptus and Jiffy in a few projects; is Jiffy completely interchangeable with msgpack, such that jiffy + leptus 1.0.0 (your proposal here) is fine?

Reply:
Sorry. I didn't meant this PR be such big. It is actually my mistake as I initially made a PR from my master branch. Now I added a lot of stuff to my master and it all appeared here. I'm going to close this PR.

As regards to your question, jiffy has been removed as an alternative to jsx (https://github.com/talentdeficit/jsx.git). leptus originally supported two different JSON libraries, in my changes it supports only one (jsx). msgpack is still supported as before.